definitions/npm/ramda_v0.21.x/flow_>=v0.28.x_<=v0.29.x/ramda_v0.21.x.js

```diff
@@ -0,0 +1,760 @@
+/* @flow */
```

Comment:
Gotcha. Yea, this bit us with 0.28.

I'd actually like to solve this at the tooling level when we deploy new versions of Flow. I'd like to make a quick little script that just runs all tests and bounds their libdefs at the new version of Flow. Then we can just run this right as we deploy each new version of Flow.

I think this will be important to make sure people aren't missing out on all the libdefs in the repo that would actually still work every time a new version of Flow goes out.

I'm going to talk with the team about it today, but in the meantime I'm happy to take on the responsibility of adding upper bounds with each of the upcoming Flow deploys.